Lib/dis.py

```diff
@@ -339,12 +348,15 @@ def _get_instructions_bytes(code, varnames=None, names=None, constants=None,
             elif op in hasfree:
                 argval, argrepr = _get_name_info(arg, cells)
             elif op == FORMAT_VALUE:
-                argval = ((None, str, repr, ascii)[arg & 0x3], bool(arg & 0x4))
-                argrepr = ('', 'str', 'repr', 'ascii')[arg & 0x3]
+                argval, argrepr = FORMAT_VALUE_CONVERTERS[arg & 0x3]
+                argval = (argval, bool(arg & 0x4))
```

Comment:
This seems like an unrelated refactoring?

Reply:
Initially I implemented the logic for MAKE_FUNCTION as a sequence of ifs. But later I seen that it can be implemented using a single constant. Since I added the MAKE_FUNCTION_FLAGS global I take the chance to move constants for FORMAT_VALUE.